paasta_tools/autoscaling/autoscaling_cluster_lib.py

```diff
@@ -1267,7 +1267,7 @@ def get_mesos_utilization_error(
         log.info("Failed to find utilization for region %s, pool %s, returning 0 error")
         return 0
 
-    log.debug(region_pool_utilization_dict)
+    log.debug(repr(region_pool_utilization_dict))
```

Comment:
this was just to make mypy happy - mypy thinks log.debug wants a string, though it's actually happy with anything repr'able.